<details><summary>The fail log I get</summary>

```
rick@nixos-asus:~/hobby/nixos-mobile-nixpkgs$ nix build .#pkgsCross.aarch64-multiplatform.asciidoc -L
src/libexpr/eval.cc:2571:23: runtime error: member call on address 0x7f30ca7f9020 which does not point to an object of type 'BaseSetting'
0x7f30ca7f9020: note: object has invalid vptr
 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00
              ^~~~~~~~~~~~~~~~~~~~~~~
              invalid vptr
src/libexpr/eval.cc:2571:53: runtime error: member call on address 0x7f30ca7f90a0 which does not point to an object of type 'BaseSetting'
0x7f30ca7f90a0: note: object has invalid vptr
 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  00 00 00 00
              ^~~~~~~~~~~~~~~~~~~~~~~
              invalid vptr
src/libutil/util.cc:1498:21: runtime error: left shift of negative value -1020460979
/nix/store/j5vhrywqmz1ixwhsmmjjxa85fpwryzh0-gcc-11.3.0/include/c++/11.3.0/bits/stl_pair.h:353:4: runtime error: load of value 25, which is not a valid value for type 'AttrType'
/nix/store/j5vhrywqmz1ixwhsmmjjxa85fpwryzh0-gcc-11.3.0/include/c++/11.3.0/bits/stl_pair.h:353:4: runtime error: load of value 33524, which is not a valid value for type 'AttrType'
/nix/store/j5vhrywqmz1ixwhsmmjjxa85fpwryzh0-gcc-11.3.0/include/c++/11.3.0/bits/stl_pair.h:353:4: runtime error: load of value 33486, which is not a valid value for type 'AttrType'
/nix/store/j5vhrywqmz1ixwhsmmjjxa85fpwryzh0-gcc-11.3.0/include/c++/11.3.0/bits/stl_pair.h:353:4: runtime error: load of value 33526, which is not a valid value for type 'AttrType'
asciidoc> Sourcing python-remove-tests-dir-hook
asciidoc> Sourcing python-remove-bin-bytecode-hook.sh
asciidoc> Sourcing setuptools-build-hook
asciidoc> Using setuptoolsBuildPhase
asciidoc> Using setuptoolsShellHook
asciidoc> Sourcing pip-install-hook
asciidoc> Using pipInstallPhase
asciidoc> Sourcing python-namespaces-hook
asciidoc> unpacking sources
asciidoc> unpacking source archive /nix/store/mnqvckww4wdfcjl41sk4q18rf8mhrknf-source
asciidoc> source root is source
asciidoc> setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/utils.py
asciidoc> patching sources
asciidoc> substituteStream(): WARNING: pattern '00:37:42' doesn't match anything in file 'CHANGELOG.adoc'
asciidoc> substituteStream(): WARNING: pattern '00:37:42' doesn't match anything in file 'tests/data/ascii-ids1-docbook5.xml'
asciidoc> substituteStream(): WARNING: pattern '00:37:42' doesn't match anything in file 'tests/data/ascii-ids1-docbook.xml'
asciidoc> substituteStream(): WARNING: pattern '00:37:42' doesn't match anything in file 'tests/data/deprecated-quotes-docbook5.xml'
asciidoc> substituteStream(): WARNING: pattern '00:37:42' doesn't match anything in file 'tests/data/deprecated-quotes-docbook.xml'
asciidoc> autoreconfPhase
asciidoc> autoreconf: export WARNINGS=
asciidoc> autoreconf: Entering directory '.'
asciidoc> autoreconf: configure.ac: not using Gettext
asciidoc> autoreconf: running: aclocal --force
asciidoc> autoreconf: configure.ac: tracing
asciidoc> autoreconf: configure.ac: not using Libtool
asciidoc> autoreconf: configure.ac: not using Intltool
asciidoc> autoreconf: configure.ac: not using Gtkdoc
asciidoc> autoreconf: running: /nix/store/6iij6rdz26gp3l8zs5iain60zhm1khzi-autoconf-2.71/bin/autoconf --force
asciidoc> autoreconf: configure.ac: not using Autoheader
asciidoc> autoreconf: configure.ac: not using Automake
asciidoc> autoreconf: './install-sh' is updated
asciidoc> autoreconf: Leaving directory '.'
asciidoc> updateAutotoolsGnuConfigScriptsPhase
asciidoc> configuring
asciidoc> configure flags: --prefix=/nix/store/y0yr9hq58d0zwhkk8xv9jhr1pgk6jwsn-asciidoc-10.2.0-aarch64-unknown-linux-gnu --build=x86_64-unknown-linux-gnu --host=aarch64-unknown-linux-gnu
asciidoc> checking for a sed that does not truncate output... /nix/store/fgg690gxg1r98b8x94c5am8wipbp81zi-gnused-4.8/bin/sed
asciidoc> checking whether ln -s works... yes
asciidoc> checking for a BSD-compatible install... /nix/store/78jhi518bs7fw478n8rzwrq0hs9pxh2f-coreutils-9.1/bin/install -c
asciidoc> configure: creating ./config.status
asciidoc> config.status: creating Makefile
asciidoc> building
asciidoc> Executing setuptoolsBuildPhase
asciidoc> running bdist_wheel
asciidoc> running build
asciidoc> running build_py
asciidoc> creating build
asciidoc> creating build/lib
asciidoc> creating build/lib/asciidoc
asciidoc> copying asciidoc/__metadata__.py -> build/lib/asciidoc
asciidoc> copying asciidoc/api.py -> build/lib/asciidoc
asciidoc> copying asciidoc/collections.py -> build/lib/asciidoc
asciidoc> copying asciidoc/__init__.py -> build/lib/asciidoc
asciidoc> copying asciidoc/__main__.py -> build/lib/asciidoc
asciidoc> copying asciidoc/asciidoc.py -> build/lib/asciidoc
asciidoc> copying asciidoc/message.py -> build/lib/asciidoc
asciidoc> copying asciidoc/exceptions.py -> build/lib/asciidoc
asciidoc> copying asciidoc/a2x.py -> build/lib/asciidoc
asciidoc> copying asciidoc/plugin.py -> build/lib/asciidoc
asciidoc> copying asciidoc/attrs.py -> build/lib/asciidoc
asciidoc> copying asciidoc/utils.py -> build/lib/asciidoc
asciidoc> creating build/lib/asciidoc/blocks
asciidoc> copying asciidoc/blocks/__init__.py -> build/lib/asciidoc/blocks
asciidoc> copying asciidoc/blocks/table.py -> build/lib/asciidoc/blocks
asciidoc> running egg_info
asciidoc> creating asciidoc.egg-info
asciidoc> writing asciidoc.egg-info/PKG-INFO
asciidoc> writing dependency_links to asciidoc.egg-info/dependency_links.txt
asciidoc> writing entry points to asciidoc.egg-info/entry_points.txt
asciidoc> writing top-level names to asciidoc.egg-info/top_level.txt
asciidoc> writing manifest file 'asciidoc.egg-info/SOURCES.txt'
asciidoc> reading manifest file 'asciidoc.egg-info/SOURCES.txt'
asciidoc> reading manifest template 'MANIFEST.in'
asciidoc> adding license file 'LICENSE'
asciidoc> writing manifest file 'asciidoc.egg-info/SOURCES.txt'
asciidoc> creating build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/asciidoc.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/docbook45.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/docbook5.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/help.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/html4.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/html5.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-ca.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-cs.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-de.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-el.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-en.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-es.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-fi.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-fr.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-hu.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-id.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-it.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-ja.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-nl.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-pl.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-pt-BR.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-ro.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-ru.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-sv.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-uk.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/lang-zh-CN.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/latex.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/slidy.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/text.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/xhtml11-quirks.conf -> build/lib/asciidoc/resources
asciidoc> copying asciidoc/resources/xhtml11.conf -> build/lib/asciidoc/resources
asciidoc> creating build/lib/asciidoc/resources/dblatex
asciidoc> copying asciidoc/resources/dblatex/asciidoc-dblatex.sty -> build/lib/asciidoc/resources/dblatex
asciidoc> copying asciidoc/resources/dblatex/asciidoc-dblatex.xsl -> build/lib/asciidoc/resources/dblatex
asciidoc> copying asciidoc/resources/dblatex/dblatex-readme.txt -> build/lib/asciidoc/resources/dblatex
asciidoc> creating build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/asciidoc-docbook-xsl.txt -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/chunked.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/common.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/epub.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/fo.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/htmlhelp.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/manpage.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/text.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> copying asciidoc/resources/docbook-xsl/xhtml.xsl -> build/lib/asciidoc/resources/docbook-xsl
asciidoc> creating build/lib/asciidoc/resources/filters
asciidoc> copying asciidoc/resources/filters/unwraplatex.py -> build/lib/asciidoc/resources/filters
asciidoc> creating build/lib/asciidoc/resources/filters/code
asciidoc> copying asciidoc/resources/filters/code/code-filter-readme.txt -> build/lib/asciidoc/resources/filters/code
asciidoc> copying asciidoc/resources/filters/code/code-filter-test.txt -> build/lib/asciidoc/resources/filters/code
asciidoc> copying asciidoc/resources/filters/code/code-filter.conf -> build/lib/asciidoc/resources/filters/code
asciidoc> copying asciidoc/resources/filters/code/code-filter.py -> build/lib/asciidoc/resources/filters/code
asciidoc> creating build/lib/asciidoc/resources/filters/graphviz
asciidoc> copying asciidoc/resources/filters/graphviz/asciidoc-graphviz-sample.txt -> build/lib/asciidoc/resources/filters/graphviz
asciidoc> copying asciidoc/resources/filters/graphviz/graphviz-filter.conf -> build/lib/asciidoc/resources/filters/graphviz
asciidoc> copying asciidoc/resources/filters/graphviz/graphviz2png.py -> build/lib/asciidoc/resources/filters/graphviz
asciidoc> creating build/lib/asciidoc/resources/filters/latex
asciidoc> copying asciidoc/resources/filters/latex/latex-filter.conf -> build/lib/asciidoc/resources/filters/latex
asciidoc> copying asciidoc/resources/filters/latex/latex2img.py -> build/lib/asciidoc/resources/filters/latex
asciidoc> creating build/lib/asciidoc/resources/filters/music
asciidoc> copying asciidoc/resources/filters/music/music-filter-test.txt -> build/lib/asciidoc/resources/filters/music
asciidoc> copying asciidoc/resources/filters/music/music-filter.conf -> build/lib/asciidoc/resources/filters/music
asciidoc> copying asciidoc/resources/filters/music/music2png.py -> build/lib/asciidoc/resources/filters/music
asciidoc> creating build/lib/asciidoc/resources/filters/source
asciidoc> copying asciidoc/resources/filters/source/source-highlight-filter-test.txt -> build/lib/asciidoc/resources/filters/source
asciidoc> copying asciidoc/resources/filters/source/source-highlight-filter.conf -> build/lib/asciidoc/resources/filters/source
asciidoc> creating build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/caution.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/example.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/home.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/important.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/next.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/note.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/prev.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/tip.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/up.png -> build/lib/asciidoc/resources/icons
asciidoc> copying asciidoc/resources/icons/warning.png -> build/lib/asciidoc/resources/icons
asciidoc> creating build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/1.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/10.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/11.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/12.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/13.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/14.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/15.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/2.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/3.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/4.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/5.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/6.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/7.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/8.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> copying asciidoc/resources/icons/callouts/9.png -> build/lib/asciidoc/resources/icons/callouts
asciidoc> creating build/lib/asciidoc/resources/javascripts
asciidoc> copying asciidoc/resources/javascripts/ASCIIMathML.js -> build/lib/asciidoc/resources/javascripts
asciidoc> copying asciidoc/resources/javascripts/LaTeXMathML.js -> build/lib/asciidoc/resources/javascripts
asciidoc> copying asciidoc/resources/javascripts/asciidoc.js -> build/lib/asciidoc/resources/javascripts
asciidoc> copying asciidoc/resources/javascripts/slidy.js -> build/lib/asciidoc/resources/javascripts
asciidoc> copying asciidoc/resources/javascripts/toc.js -> build/lib/asciidoc/resources/javascripts
asciidoc> creating build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/asciidoc.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/docbook-xsl.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/pygments.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/slidy.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/toc2.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> copying asciidoc/resources/stylesheets/xhtml11-quirks.css -> build/lib/asciidoc/resources/stylesheets
asciidoc> creating build/lib/asciidoc/resources/themes
asciidoc> creating build/lib/asciidoc/resources/themes/flask
asciidoc> copying asciidoc/resources/themes/flask/flask.css -> build/lib/asciidoc/resources/themes/flask
asciidoc> creating build/lib/asciidoc/resources/themes/volnitsky
asciidoc> copying asciidoc/resources/themes/volnitsky/volnitsky.css -> build/lib/asciidoc/resources/themes/volnitsky
asciidoc> /nix/store/2nvsyjhyg74af3g7dg4ghzmnd2844mj2-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
asciidoc>   warnings.warn(
asciidoc> installing to build/bdist.linux-aarch64/wheel
asciidoc> running install
asciidoc> running install_lib
asciidoc> creating build/bdist.linux-aarch64
asciidoc> creating build/bdist.linux-aarch64/wheel
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/javascripts/slidy.js -> build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/javascripts/toc.js -> build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/javascripts/ASCIIMathML.js -> build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/javascripts/LaTeXMathML.js -> build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/javascripts/asciidoc.js -> build/bdist.linux-aarch64/wheel/asciidoc/resources/javascripts
asciidoc> copying build/lib/asciidoc/resources/lang-hu.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-uk.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-sv.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-pt-BR.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/text.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-fi.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/xhtml11-quirks.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-nl.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-it.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-ca.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-id.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/html4.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/pygments.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/docbook-xsl.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/asciidoc.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/xhtml11-quirks.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/slidy.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/stylesheets/toc2.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/stylesheets
asciidoc> copying build/lib/asciidoc/resources/lang-fr.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/html5.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-de.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/xhtml11.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/asciidoc.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/up.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/3.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/14.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/2.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/1.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/6.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/11.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/10.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/7.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/12.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/9.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/8.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/13.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/15.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/4.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/callouts/5.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons/callouts
asciidoc> copying build/lib/asciidoc/resources/icons/prev.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/caution.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/important.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/example.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/warning.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/note.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/tip.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/next.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/icons/home.png -> build/bdist.linux-aarch64/wheel/asciidoc/resources/icons
asciidoc> copying build/lib/asciidoc/resources/help.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-el.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-en.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-cs.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-zh-CN.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/dblatex
asciidoc> copying build/lib/asciidoc/resources/dblatex/asciidoc-dblatex.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/dblatex
asciidoc> copying build/lib/asciidoc/resources/dblatex/dblatex-readme.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/dblatex
asciidoc> copying build/lib/asciidoc/resources/dblatex/asciidoc-dblatex.sty -> build/bdist.linux-aarch64/wheel/asciidoc/resources/dblatex
asciidoc> copying build/lib/asciidoc/resources/lang-ru.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/epub.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/xhtml.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/htmlhelp.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/text.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/fo.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/chunked.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/common.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/asciidoc-docbook-xsl.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/docbook-xsl/manpage.xsl -> build/bdist.linux-aarch64/wheel/asciidoc/resources/docbook-xsl
asciidoc> copying build/lib/asciidoc/resources/lang-es.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/themes
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/themes/flask
asciidoc> copying build/lib/asciidoc/resources/themes/flask/flask.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/themes/flask
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/themes/volnitsky
asciidoc> copying build/lib/asciidoc/resources/themes/volnitsky/volnitsky.css -> build/bdist.linux-aarch64/wheel/asciidoc/resources/themes/volnitsky
asciidoc> copying build/lib/asciidoc/resources/docbook5.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/slidy.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/code
asciidoc> copying build/lib/asciidoc/resources/filters/code/code-filter-readme.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/code
asciidoc> copying build/lib/asciidoc/resources/filters/code/code-filter.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/code
asciidoc> copying build/lib/asciidoc/resources/filters/code/code-filter.py -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/code
asciidoc> copying build/lib/asciidoc/resources/filters/code/code-filter-test.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/code
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/latex
asciidoc> copying build/lib/asciidoc/resources/filters/latex/latex-filter.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/latex
asciidoc> copying build/lib/asciidoc/resources/filters/latex/latex2img.py -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/latex
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/music
asciidoc> copying build/lib/asciidoc/resources/filters/music/music-filter-test.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/music
asciidoc> copying build/lib/asciidoc/resources/filters/music/music2png.py -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/music
asciidoc> copying build/lib/asciidoc/resources/filters/music/music-filter.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/music
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/source
asciidoc> copying build/lib/asciidoc/resources/filters/source/source-highlight-filter.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/source
asciidoc> copying build/lib/asciidoc/resources/filters/source/source-highlight-filter-test.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/source
asciidoc> copying build/lib/asciidoc/resources/filters/unwraplatex.py -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/graphviz
asciidoc> copying build/lib/asciidoc/resources/filters/graphviz/graphviz2png.py -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/graphviz
asciidoc> copying build/lib/asciidoc/resources/filters/graphviz/asciidoc-graphviz-sample.txt -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/graphviz
asciidoc> copying build/lib/asciidoc/resources/filters/graphviz/graphviz-filter.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources/filters/graphviz
asciidoc> copying build/lib/asciidoc/resources/lang-ro.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-pl.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/lang-ja.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/docbook45.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/resources/latex.conf -> build/bdist.linux-aarch64/wheel/asciidoc/resources
asciidoc> copying build/lib/asciidoc/__metadata__.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/api.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/collections.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/__init__.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc/blocks
asciidoc> copying build/lib/asciidoc/blocks/__init__.py -> build/bdist.linux-aarch64/wheel/asciidoc/blocks
asciidoc> copying build/lib/asciidoc/blocks/table.py -> build/bdist.linux-aarch64/wheel/asciidoc/blocks
asciidoc> copying build/lib/asciidoc/__main__.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/asciidoc.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/message.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/exceptions.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/a2x.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/plugin.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/attrs.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> copying build/lib/asciidoc/utils.py -> build/bdist.linux-aarch64/wheel/asciidoc
asciidoc> running install_egg_info
asciidoc> Copying asciidoc.egg-info to build/bdist.linux-aarch64/wheel/asciidoc-10.2.0-py3.10.egg-info
asciidoc> running install_scripts
asciidoc> adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
asciidoc> creating build/bdist.linux-aarch64/wheel/asciidoc-10.2.0.dist-info/WHEEL
asciidoc> creating 'dist/asciidoc-10.2.0-py3-none-any.whl' and adding 'build/bdist.linux-aarch64/wheel' to it
asciidoc> adding 'asciidoc/__init__.py'
asciidoc> adding 'asciidoc/__main__.py'
asciidoc> adding 'asciidoc/__metadata__.py'
asciidoc> adding 'asciidoc/a2x.py'
asciidoc> adding 'asciidoc/api.py'
asciidoc> adding 'asciidoc/asciidoc.py'
asciidoc> adding 'asciidoc/attrs.py'
asciidoc> adding 'asciidoc/collections.py'
asciidoc> adding 'asciidoc/exceptions.py'
asciidoc> adding 'asciidoc/message.py'
asciidoc> adding 'asciidoc/plugin.py'
asciidoc> adding 'asciidoc/utils.py'
asciidoc> adding 'asciidoc/blocks/__init__.py'
asciidoc> adding 'asciidoc/blocks/table.py'
asciidoc> adding 'asciidoc/resources/asciidoc.conf'
asciidoc> adding 'asciidoc/resources/docbook45.conf'
asciidoc> adding 'asciidoc/resources/docbook5.conf'
asciidoc> adding 'asciidoc/resources/help.conf'
asciidoc> adding 'asciidoc/resources/html4.conf'
asciidoc> adding 'asciidoc/resources/html5.conf'
asciidoc> adding 'asciidoc/resources/lang-ca.conf'
asciidoc> adding 'asciidoc/resources/lang-cs.conf'
asciidoc> adding 'asciidoc/resources/lang-de.conf'
asciidoc> adding 'asciidoc/resources/lang-el.conf'
asciidoc> adding 'asciidoc/resources/lang-en.conf'
asciidoc> adding 'asciidoc/resources/lang-es.conf'
asciidoc> adding 'asciidoc/resources/lang-fi.conf'
asciidoc> adding 'asciidoc/resources/lang-fr.conf'
asciidoc> adding 'asciidoc/resources/lang-hu.conf'
asciidoc> adding 'asciidoc/resources/lang-id.conf'
asciidoc> adding 'asciidoc/resources/lang-it.conf'
asciidoc> adding 'asciidoc/resources/lang-ja.conf'
asciidoc> adding 'asciidoc/resources/lang-nl.conf'
asciidoc> adding 'asciidoc/resources/lang-pl.conf'
asciidoc> adding 'asciidoc/resources/lang-pt-BR.conf'
asciidoc> adding 'asciidoc/resources/lang-ro.conf'
asciidoc> adding 'asciidoc/resources/lang-ru.conf'
asciidoc> adding 'asciidoc/resources/lang-sv.conf'
asciidoc> adding 'asciidoc/resources/lang-uk.conf'
asciidoc> adding 'asciidoc/resources/lang-zh-CN.conf'
asciidoc> adding 'asciidoc/resources/latex.conf'
asciidoc> adding 'asciidoc/resources/slidy.conf'
asciidoc> adding 'asciidoc/resources/text.conf'
asciidoc> adding 'asciidoc/resources/xhtml11-quirks.conf'
asciidoc> adding 'asciidoc/resources/xhtml11.conf'
asciidoc> adding 'asciidoc/resources/dblatex/asciidoc-dblatex.sty'
asciidoc> adding 'asciidoc/resources/dblatex/asciidoc-dblatex.xsl'
asciidoc> adding 'asciidoc/resources/dblatex/dblatex-readme.txt'
asciidoc> adding 'asciidoc/resources/docbook-xsl/asciidoc-docbook-xsl.txt'
asciidoc> adding 'asciidoc/resources/docbook-xsl/chunked.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/common.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/epub.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/fo.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/htmlhelp.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/manpage.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/text.xsl'
asciidoc> adding 'asciidoc/resources/docbook-xsl/xhtml.xsl'
asciidoc> adding 'asciidoc/resources/filters/unwraplatex.py'
asciidoc> adding 'asciidoc/resources/filters/code/code-filter-readme.txt'
asciidoc> adding 'asciidoc/resources/filters/code/code-filter-test.txt'
asciidoc> adding 'asciidoc/resources/filters/code/code-filter.conf'
asciidoc> adding 'asciidoc/resources/filters/code/code-filter.py'
asciidoc> adding 'asciidoc/resources/filters/graphviz/asciidoc-graphviz-sample.txt'
asciidoc> adding 'asciidoc/resources/filters/graphviz/graphviz-filter.conf'
asciidoc> adding 'asciidoc/resources/filters/graphviz/graphviz2png.py'
asciidoc> adding 'asciidoc/resources/filters/latex/latex-filter.conf'
asciidoc> adding 'asciidoc/resources/filters/latex/latex2img.py'
asciidoc> adding 'asciidoc/resources/filters/music/music-filter-test.txt'
asciidoc> adding 'asciidoc/resources/filters/music/music-filter.conf'
asciidoc> adding 'asciidoc/resources/filters/music/music2png.py'
asciidoc> adding 'asciidoc/resources/filters/source/source-highlight-filter-test.txt'
asciidoc> adding 'asciidoc/resources/filters/source/source-highlight-filter.conf'
asciidoc> adding 'asciidoc/resources/icons/caution.png'
asciidoc> adding 'asciidoc/resources/icons/example.png'
asciidoc> adding 'asciidoc/resources/icons/home.png'
asciidoc> adding 'asciidoc/resources/icons/important.png'
asciidoc> adding 'asciidoc/resources/icons/next.png'
asciidoc> adding 'asciidoc/resources/icons/note.png'
asciidoc> adding 'asciidoc/resources/icons/prev.png'
asciidoc> adding 'asciidoc/resources/icons/tip.png'
asciidoc> adding 'asciidoc/resources/icons/up.png'
asciidoc> adding 'asciidoc/resources/icons/warning.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/1.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/10.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/11.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/12.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/13.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/14.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/15.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/2.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/3.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/4.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/5.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/6.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/7.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/8.png'
asciidoc> adding 'asciidoc/resources/icons/callouts/9.png'
asciidoc> adding 'asciidoc/resources/javascripts/ASCIIMathML.js'
asciidoc> adding 'asciidoc/resources/javascripts/LaTeXMathML.js'
asciidoc> adding 'asciidoc/resources/javascripts/asciidoc.js'
asciidoc> adding 'asciidoc/resources/javascripts/slidy.js'
asciidoc> adding 'asciidoc/resources/javascripts/toc.js'
asciidoc> adding 'asciidoc/resources/stylesheets/asciidoc.css'
asciidoc> adding 'asciidoc/resources/stylesheets/docbook-xsl.css'
asciidoc> adding 'asciidoc/resources/stylesheets/pygments.css'
asciidoc> adding 'asciidoc/resources/stylesheets/slidy.css'
asciidoc> adding 'asciidoc/resources/stylesheets/toc2.css'
asciidoc> adding 'asciidoc/resources/stylesheets/xhtml11-quirks.css'
asciidoc> adding 'asciidoc/resources/themes/flask/flask.css'
asciidoc> adding 'asciidoc/resources/themes/volnitsky/volnitsky.css'
asciidoc> adding 'asciidoc-10.2.0.dist-info/LICENSE'
asciidoc> adding 'asciidoc-10.2.0.dist-info/METADATA'
asciidoc> adding 'asciidoc-10.2.0.dist-info/WHEEL'
asciidoc> adding 'asciidoc-10.2.0.dist-info/entry_points.txt'
asciidoc> adding 'asciidoc-10.2.0.dist-info/top_level.txt'
asciidoc> adding 'asciidoc-10.2.0.dist-info/RECORD'
asciidoc> removing build/bdist.linux-aarch64/wheel
asciidoc> python3 -m asciidoc.a2x -f manpage doc/asciidoc.1.txt
asciidoc> make: python3: No such file or directory
asciidoc> make: *** [Makefile:56: doc/asciidoc.1] Error 127
```

</details>

For commit c54662430287d0f8d3fc99dea90cbbe05992f8cd on this branch.

--- a/pkgs/development/libraries/libjxl/default.nix
+++ b/pkgs/development/libraries/libjxl/default.nix
@@ -47,6 +47,17 @@ stdenv.mkDerivation rec {
       url = "https://github.com/libjxl/libjxl/commit/204f87a5e4d684544b13900109abf040dc0b402b.patch";
       sha256 = "sha256-DoAaYWLmQ+R9GZbHMTYGe0gBL9ZesgtB+2WhmbARna8=";
     })
+
+    # fix build with asciidoc wrapped in shell script
+    (fetchpatch {
+      url = "https://github.com/libjxl/libjxl/commit/b8ec58c58c6281987f42ebec892f513824c0cc0e.patch";
+      hash = "sha256-g8U+YVhLfgSHJ+PWJgvVOI66+FElJSC8IgSRodNnsMw=";
+    })
+    (fetchpatch {
+      url = "https://github.com/libjxl/libjxl/commit/ca8e276aacf63a752346a6a44ba673b0af993237.patch";
+      excludes = [ "AUTHORS" ];
+      hash = "sha256-9VXy1LdJ0JhYbCGPNMySpnGLBxUrr8BYzE+oU3LnUGw=";
+    })
   ];
 
   nativeBuildInputs = [

--- a/pkgs/tools/typesetting/asciidoc/default.nix
+++ b/pkgs/tools/typesetting/asciidoc/default.nix
@@ -1,5 +1,6 @@
 { fetchurl, lib, stdenv, python3
 , fetchFromGitHub, autoreconfHook
+, installShellFiles
 , enableStandardFeatures ? false
 , sourceHighlight ? null
 , highlight ? null
@@ -17,8 +18,7 @@
 , docbook_xsl_ns ? null
 , docbook_xsl ? null
 , fop ? null
-# TODO: Package this:
-#, epubcheck ? null
+, epubcheck ? null
 , gnused ? null
 , coreutils ? null
 
@@ -61,8 +61,7 @@ assert enableStandardFeatures ->
   docbook_xsl_ns != null &&
   docbook_xsl != null &&
   (fop != null || !enableJava) &&
-# TODO: Package this:
-#  epubcheck != null &&
+  epubcheck != null &&
   gnused != null &&
   coreutils != null;
 
@@ -144,26 +143,24 @@ let
     sha256 = "08ya4bskygzqkfqwjllpg31qc5k08xp2k78z9b2480g8y57bfy10";
   };
 
-in
-
-stdenv.mkDerivation rec {
+in python3.pkgs.buildPythonApplication rec {
   pname = "asciidoc"
     + lib.optionalString enableStandardFeatures "-full"
     + lib.optionalString enableExtraPlugins "-with-plugins";
-  version = "9.1.0";
+  version = "10.2.0";
 
-  # Note: a substitution to improve reproducibility should be updated once 10.0.0 is
-  # released. See the comment in `patchPhase` for more information.
   src = fetchFromGitHub {
-    owner = "asciidoc";
-    repo = "asciidoc-py3";
+    owner = "asciidoc-py";
+    repo = "asciidoc-py";
     rev = version;
-    sha256 = "1clf1axkns23wfmh48xfspzsnw04pjh4mq1pshpzvj0cwxhz0yaq";
+    hash = "sha256-TqC0x9xB6e2d6Wc9bgnlqgZVOmYHmUUKfE/CKAiEtag=";
   };
 
-  strictDeps = true;
-  nativeBuildInputs = [ python3 unzip autoreconfHook ];
-  buildInputs = [ python3 ];
+  nativeBuildInputs = [
+    autoreconfHook
+    installShellFiles
+    unzip
+  ];
 
   # install filters early, so their shebangs are patched too
   postPatch = with lib; ''
@@ -230,22 +227,22 @@ stdenv.mkDerivation rec {
         -e "s|twopi|${graphviz}/bin/twopi|g" \
         -e "s|circo|${graphviz}/bin/circo|g" \
         -e "s|fdp|${graphviz}/bin/fdp|g" \
-        -i "filters/graphviz/graphviz2png.py"
+        -i "asciidoc/resources/filters/graphviz/graphviz2png.py"
 
     sed -e "s|run('latex|run('${texlive}/bin/latex|g" \
         -e "s|cmd = 'dvipng'|cmd = '${texlive}/bin/dvipng'|g" \
         -e "s|cmd = 'dvisvgm'|cmd = '${texlive}/bin/dvisvgm'|g" \
-        -i "filters/latex/latex2img.py"
+        -i "asciidoc/resources/filters/latex/latex2img.py"
 
     sed -e "s|run('abc2ly|run('${lilypond}/bin/abc2ly|g" \
         -e "s|run('lilypond|run('${lilypond}/bin/lilypond|g" \
         -e "s|run('convert|run('${imagemagick.out}/bin/convert|g" \
-        -i "filters/music/music2png.py"
+        -i "asciidoc/resources/filters/music/music2png.py"
 
     sed -e 's|filter="source-highlight|filter="${sourceHighlight}/bin/source-highlight|' \
         -e 's|filter="highlight|filter="${highlight}/bin/highlight|' \
         -e 's|filter="pygmentize|filter="${pygments}/bin/pygmentize|' \
-        -i "filters/source/source-highlight-filter.conf"
+        -i "asciidoc/resources/filters/source/source-highlight-filter.conf"
 
     # ENV is custom environment passed to programs that a2x invokes. Here we
     # use it to work around an impurity in the tetex package; tetex tools
@@ -260,55 +257,45 @@ stdenv.mkDerivation rec {
         -e "s|^W3M =.*|W3M = '${w3m}/bin/w3m'|" \
         -e "s|^LYNX =.*|LYNX = '${lynx}/bin/lynx'|" \
         -e "s|^XMLLINT =.*|XMLLINT = '${libxml2.bin}/bin/xmllint'|" \
-        -e "s|^EPUBCHECK =.*|EPUBCHECK = 'nixpkgs_is_missing_epubcheck'|" \
-        -i a2x.py
+        -e "s|^EPUBCHECK =.*|EPUBCHECK = '${epubcheck}/bin/epubcheck'|" \
+        -i asciidoc/a2x.py
   '' else ''
     sed -e "s|^ENV =.*|ENV = dict(XML_CATALOG_FILES='${docbook_xml_dtd_45}/xml/dtd/docbook/catalog.xml ${docbook_xsl_ns}/xml/xsl/docbook/catalog.xml ${docbook_xsl}/xml/xsl/docbook/catalog.xml')|" \
         -e "s|^XSLTPROC =.*|XSLTPROC = '${libxslt.bin}/bin/xsltproc'|" \
         -e "s|^XMLLINT =.*|XMLLINT = '${libxml2.bin}/bin/xmllint'|" \
-        -i a2x.py
+        -i asciidoc/a2x.py
   '') + ''
-    patchShebangs --host \
-      asciidoc.py \
-      a2x.py \
-      tests/testasciidoc.py \
-      filters/code/code-filter.py \
-      filters/latex/latex2img.py \
-      filters/music/music2png.py \
-      filters/unwraplatex.py \
-      filters/graphviz/graphviz2png.py
-
-    # Hardcode the path to its own asciidoc.
-    # This helps with cross-compilation.
-    substituteInPlace a2x.py \
-      --replace "find_executable(ASCIIDOC)" "'${placeholder "out"}/bin/asciidoc'"
-
-    # Note: this substitution will not work in the planned 10.0.0 release:
-    #
-    # https://github.com/asciidoc/asciidoc-py3/commit/dfffda23381014481cd13e8e9d8f131e1f93f08a
-    #
-    # Update this substitution to:
-    #
-    # --replace "python3 -m asciidoc.a2x" "python3 -m asciidoc.a2x -a revdate=01/01/1980"
-    substituteInPlace Makefile.in \
-      --replace "python3 a2x.py" "python3 a2x.py -a revdate=01/01/1980"
-
     # Fix tests
     for f in $(grep -R --files-with-matches "2002-11-25") ; do
-      substituteInPlace $f --replace "2002-11-25" "1970-01-01"
-      substituteInPlace $f --replace "00:37:42" "00:00:01"
+      substituteInPlace $f --replace "2002-11-25" "1980-01-02"
+      substituteInPlace $f --replace "00:37:42" "00:00:00"
     done
   '' + lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
     # We want to use asciidoc from the build platform to build the documentation.
     substituteInPlace Makefile.in \
-      --replace "python3 a2x.py" "python3 ${buildPackages.asciidoc}/bin/a2x.py"
+      --replace "python3 -m asciidoc.a2x" "${buildPackages.asciidoc}/bin/a2x"
   '';
 
-  preInstall = "mkdir -p $out/etc/vim";
-  makeFlags = lib.optional stdenv.isCygwin "DESTDIR=/.";
+  postBuild = ''
+    make manpages
+  '';
 
-  checkInputs = [ sourceHighlight ];
-  doCheck = true;
+  postInstall = ''
+    installManPage doc/asciidoc.1 doc/a2x.1 doc/testasciidoc.1
+  '';
+
+  checkInputs = with python3.pkgs; [
+    pytest
+    pytest-mock
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    make test
+
+    runHook postCheck
+  '';
 
   meta = with lib; {
     description = "Text-based document generation system";
@@ -325,9 +312,10 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [
       fromSource
     ] ++ lib.optional _enableDitaaFilter binaryBytecode;
-    homepage = "http://www.methods.co.nz/asciidoc/";
+    homepage = "https://asciidoc-py.github.io/";
+    changelog = "https://github.com/asciidoc-py/asciidoc-py/blob/${src.rev}/CHANGELOG.adoc";
     license = licenses.gpl2Plus;
     platforms = platforms.unix;
-    maintainers = [ maintainers.bjornfor ];
+    maintainers = with maintainers; [ bjornfor dotlambda ];
   };
 }

--- a/pkgs/tools/typesetting/asciidoc/default.nix
+++ b/pkgs/tools/typesetting/asciidoc/default.nix
@@ -2,38 +2,38 @@
 , fetchFromGitHub, autoreconfHook
 , installShellFiles
 , enableStandardFeatures ? false
-, sourceHighlight ? null
-, highlight ? null
-, pygments ? null
-, graphviz ? null
-, texlive ? null
-, dblatexFull ? null
-, libxslt ? null
-, w3m ? null
-, lynx ? null
-, imagemagick ? null
-, lilypond ? null
-, libxml2 ? null
-, docbook_xml_dtd_45 ? null
-, docbook_xsl_ns ? null
-, docbook_xsl ? null
-, fop ? null
-, epubcheck ? null
-, gnused ? null
-, coreutils ? null
+, sourceHighlight
+, highlight
+, pygments
+, graphviz
+, texlive
+, dblatexFull
+, libxslt
+, w3m
+, lynx
+, imagemagick
+, lilypond
+, libxml2
+, docbook_xml_dtd_45
+, docbook_xsl_ns
+, docbook_xsl
+, fop
+, epubcheck
+, gnused
+, coreutils
 
 # if true, enable all the below filters and backends
 , enableExtraPlugins ? false
 
 # unzip is needed to extract filter and backend plugins
-, unzip ? null
+, unzip
 # filters
-, enableDitaaFilter ? false, jre ? null
-, enableMscgenFilter ? false, mscgen ? null
-, enableDiagFilter ? false, blockdiag ? null, seqdiag ? null, actdiag ? null, nwdiag ? null
-, enableQrcodeFilter ? false, qrencode ? null
-, enableMatplotlibFilter ? false, matplotlib ? null, numpy ? null
-, enableAafigureFilter ? false, aafigure ? null, recursivePthLoader ? null
+, enableDitaaFilter ? false, jre
+, enableMscgenFilter ? false, mscgen
+, enableDiagFilter ? false, blockdiag, seqdiag, actdiag, nwdiag
+, enableQrcodeFilter ? false, qrencode
+, enableMatplotlibFilter ? false, matplotlib, numpy
+, enableAafigureFilter ? false, aafigure, recursivePthLoader
 # backends
 , enableDeckjsBackend ? false
 , enableOdfBackend ? false
@@ -43,37 +43,6 @@
 
 , buildPackages
 }:
-
-assert enableStandardFeatures ->
-  sourceHighlight != null &&
-  highlight != null &&
-  pygments != null &&
-  graphviz != null &&
-  texlive != null &&
-  dblatexFull != null &&
-  libxslt != null &&
-  w3m != null &&
-  lynx != null &&
-  imagemagick != null &&
-  lilypond != null &&
-  libxml2 != null &&
-  docbook_xml_dtd_45 != null &&
-  docbook_xsl_ns != null &&
-  docbook_xsl != null &&
-  (fop != null || !enableJava) &&
-  epubcheck != null &&
-  gnused != null &&
-  coreutils != null;
-
-# filters
-assert enableExtraPlugins || enableDitaaFilter || enableMscgenFilter || enableDiagFilter || enableQrcodeFilter || enableAafigureFilter -> unzip != null;
-assert (enableExtraPlugins && enableJava) || enableDitaaFilter -> jre != null;
-assert enableExtraPlugins || enableMscgenFilter -> mscgen != null;
-assert enableExtraPlugins || enableDiagFilter -> blockdiag != null && seqdiag != null && actdiag != null && nwdiag != null;
-assert enableExtraPlugins || enableMatplotlibFilter -> matplotlib != null && numpy != null;
-assert enableExtraPlugins || enableAafigureFilter -> aafigure != null && recursivePthLoader != null;
-# backends
-assert enableExtraPlugins || enableDeckjsBackend || enableOdfBackend -> unzip != null;
 
 let
 

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -4470,21 +4470,17 @@ with pkgs;
   arpoison = callPackage ../tools/networking/arpoison { };
 
   asciidoc = callPackage ../tools/typesetting/asciidoc {
-    inherit (python3.pkgs) matplotlib numpy aafigure recursivePthLoader;
+    inherit (python3.pkgs) pygments matplotlib numpy aafigure recursivePthLoader;
+    texlive = texlive.combine { inherit (texlive) scheme-minimal dvipng; };
+    w3m = w3m-batch;
     enableStandardFeatures = false;
   };
 
   asciidoc-full = asciidoc.override {
-    inherit (python3.pkgs) pygments;
-    texlive = texlive.combine { inherit (texlive) scheme-minimal dvipng; };
-    w3m = w3m-batch;
     enableStandardFeatures = true;
   };
 
   asciidoc-full-with-plugins = asciidoc.override {
-    inherit (python3.pkgs) pygments;
-    texlive = texlive.combine { inherit (texlive) scheme-minimal dvipng; };
-    w3m = w3m-batch;
     enableStandardFeatures = true;
     enableExtraPlugins = true;
   };

